src/containers/couresCarousel/index.js

```diff
@@ -20,7 +20,7 @@ const responsive = {
   730: {
     items: 2,
   },
-  350: {
+  370: {
```

Comment:
Look, we have such endpoints for CSS.
```
$width-xs: 320px;
$width-sm: 576px;
$width-md: 768px;
$width-lg: 992px;
$width-xl: 1200px;
```

Maybe it useful to have such endpoints into the carousel.